src/components/Button.js

```diff
@@ -21,11 +25,41 @@ const Button = React.createClass({
 
   getDefaultProps () {
     return {
+      isActive: false,
       primaryColor: StyleConstants.Colors.PRIMARY,
       type: 'primary'
     };
   },
 
+  _renderButtonContent () {
+    const spinnerStyles = {
+      verticalAlign: 'initial',
+      marginTop: -6,
+      marginBottom: -5,
+      marginLeft: -5,
+      marginRight: -5,
+      padding: !this.props.actionText && this.props.children ? 3 : 0
+    };
+
+    const actionTextStyles = {
+      display: 'inline-block',
+      paddingLeft: 10
+    };
```

Comment:
We've established a new pattern for component styles.  We've started adding a `styles` function at the bottom of the component below the main render function.  This new `styles` function just return an object that has all the styles.  Then in other functions if we need those styles we just make a call like so.

```javascript
const styles = this.styles()'

return <div style={styles.divStyle} />
```

Since your touching the styles in the render function and adding these here as well I would suggest going ahead and moving all the styles to a styles function to match the new pattern.  Come bug me if you have questions.